wireguard/wg-init

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
```

Comment:
Maybe `wg-autostart` or `wg-autostart-ctl` is a more descriptive name?

Reply:
I think the idea of this script is really nice. I'd like a comment in here that says that the non-systemctl method is for 6.x so we know it can be deleted once we drop support for DSM 6.

Reply:
As suggested, I will change the name to `wg-autostart`, as it says more about the script than `wg-init`.